gk/main.c

```diff
@@ -1175,6 +1245,384 @@ gk_conf_put(struct gk_config *gk_conf)
 	return 0;
 }
 
+static int
+gk_lpm_add_ipv4_routes(
+	struct rte_lpm *lpm, struct ipv4_lpm_route *routes,
+	unsigned int num_routes, struct gk_lpm *ltbl)
+{
+	int ret = 0;
+	unsigned int i;
+	for (i = 0; i < num_routes; i++) {
+		ret = rte_lpm_add(lpm, routes[i].ip,
```

Comment:
Yes, it's converted before calling this function. As suggested below, I will move the conversion into this function.